aerofiles/seeyou/reader.py

```diff
@@ -193,3 +221,101 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_task_options(self, fields):
+        if not fields[0] == "Options":
+            return
+
+        task_options = {
+            'no_start': None,
+            'task_time': None,
+            'wp_dis': False,
+            'near_dis': None,
+            'near_alt': None,
+            'min_dis': False,
+            'random_order': False,
+            'max_pts': None,
+            'before_pts': None,
+            'after_pts': None,
+            'bonus': None
+        }
+
+        for field in fields[1:]:
+            field_type = field.split("=")[0]
+            field_entry = field.split("=")[1]
```

Comment:
could you extract `field.split('=')` into a local variable to remove/merge the duplicate `split()` call?